### DISCUSSION TITLE
Anyone got the camera feeds working with Nest cameras?

### DISCUSSION THREAD

**Discussion by TheOpsMgr** [matt8707/ha-fusion#416] [Q&A] [answered]
Anyone got the camera feeds working with Nest cameras?

I get an image when I edit the UI to set them up...

![image](https://github.com/matt8707/ha-fusion/assets/3353242/05bd51d9-8151-40ab-af3a-cf78cc6ae5d5)

But just black boxes when I view them in the dashboard

![image](https://github.com/matt8707/ha-fusion/assets/3353242/bcb50d08-c0e5-475b-9d0a-64c2ef4c6526)

Looks like http://homeassistant.local:8123/api/**camera_proxy**/camera.backyard?token=<token redacted> loads but http://homeassistant.local:8123/api/**camera_proxy_stream**/camera.backyard?token=<token redacted>

**Comment by mucki12** [comment, marked as answer]
Same problem?
https://github.com/matt8707/ha-fusion/issues/369

**Reply by TheOpsMgr** [reply]
Yes. Exactly.

Sorry, I was in a hurry before heading away on holidays and didn't have time to search for bugs. Bad form!

S.

**Comment by matt8707** [comment]
https://github.com/matt8707/ha-fusion/releases/tag/2024.5.0
